src/Reporter.ts

```diff
@@ -129,6 +134,11 @@ class Reporter {
                     const socket: net.Socket = net.createConnection(options.port as number, options.hostname);
                     socket.setNoDelay(true);
                     socket.setKeepAlive(true);
+
+                    socket.setTimeout(REPORTER_HTTP_TIMEOUT, () => {
+                        ThundraLogger.debug('<Reporter> Reporter socket timeout');
```

Comment:
I think, log level should be at error level and log message should mention about possible public internet connectivity issue

Reply:
```js
ThundraLogger.error('<Reporter> Reporter socket timeout. Please be sure that your application has access to public internet.');
```